l3build-check.lua

```diff
@@ -574,6 +574,7 @@ function runcheck(name, hide)
   if options["engine"] then
     checkengines = options["engine"]
   end
+  local failedengines = {}
```

Comment:
failed_engines

Reply:
`failedengines` is more consistent with similar names like `checkengines`, `asciiengines`, etc.

Reply:
Consistency only applies to similar objects.
`checkengines` and friends are global variables also available from the build.lua.
Your `failedengines` has a completely different semantics, and is local.

Reply:
Technically it is a local variable here :stuck_out_tongue_winking_eye: More seriously, we use both naming schemes in multiple places for local variables and in this situation, especially given that it's basically the subset of `checkengines` which failed, I think that consistency with `checkengines` is more important.